great-idea-website/css/index.css

```diff
@@ -63,4 +63,203 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+
+
+/* Makes Everything Centered */
+
+.container {
+	width: 80%;
+	margin: auto;
+	overflow: hidden;
+	
+ }
+ 
+ 
+ 
+ 
+ /* Header, Nav, and Logo */
+ 
+ header {
+	 position: relative;
+	 margin-top: 20px;
+	 height: 120px;
+	
+	 
+ }
+ 
+ header nav {
+	 display: inline-block;
+	 position: aboslute;
+	 line-height: 40px;
+	 position: fixed;
+	 
+	
+	 
+ }
+ 
+ header nav a {
+	 text-decoration: none;
+	 color: black;
+	 margin: 20px;
+	 border: 1px solid gray;
+	 border-radius: 5px;
+	 padding: 10px 20px;
+	 font-family: 'Text Me One', sans-serif;
+	 transition: 0.2s ease-in;	
+ }
+ 
```

Comment:
The nav bar looked very stylish, and I like the fixed navbar that stays on top of the window as you scroll down the page. If you wrap the whole navbar in a container div, and set its background to white and position: fixed, the nav buttons will keep their white background as you scroll down the page.